raw_map/src/transform/find_short_roads.rs

```diff
@@ -105,10 +105,8 @@ impl RawMap {
             {
                 continue;
             }
-            if let Ok((pl, _)) = road.untrimmed_road_geometry() {
-                if pl.length() <= threshold {
-                    results.push(*id);
-                }
+            if road.untrimmed_road_geometry().0.length() <= threshold {
```

Comment:
Example of benefits -- some of these transformation functions happened to hit errors, so I threw in skips here and there without rhyme or reason. It's better to make sure all `RawRoads` have a legal `PolyLine` as a center up-front.